CHANGELOG.md

```diff
@@ -9,6 +9,39 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.5.0 - 2019-03-25
+
+### Added
+
+- Add "networkPolicyOnly" as a new "encapsulation mode".
+- Support for running Antrea in [EKS] and [GKE] clusters; refer to the documentation.
+- New antctl "get" commands:
+   * in "agent mode": addressgroup, agentinfo, appliedtogroup, networkpolicy, podinterface
+   * in "controller mode": addressgroup, appliedtogroup, controllerinfo, networkpolicy
+- Support for a user-friendly "table" output format for antctl "get" commands.
+- Add health checks to Antrea components by leveraging the apiserver /healthz endpoint (both the Antrea Agent and Controller are running an apiserver).
+- Add documentation for troubleshooting connection issues to the Antrea Agent or Controller apiserver.
+- Ship antctl binaries as part of each release for different OS / CPU combinations: antctl-linux-x86_64, antctl-linux-arm, antctl-linux-arm64, antctl-windows-x86_64.exe, antctl-darwin-x86_64.
+- Add documentation for antctl installation and usage.
+
+### Changed
+
+- Refactor of antctl: most notable change is that the Antrea Agent now runs its own apiserver which the antctl CLI can connect to.
+- Improve NetworkPolicy logging; in particular an Agent now logs (by default) a message when it receives a new NetworkPolicy that needs to be implemented locally.
+- Upgrade OVS to version 2.13.0, which comes with userspace datapath improvements useful when running Antrea in Kind.
+- Refactor of the Antrea "route" interface which manages routes and firewall rules on the Node; besides improvements to the code base, the following changes were made:
```

Comment:
Feel no need to talk about code refactoring. But up to you.

Reply:
I removed the bit about refactoring, but kept the 2 iptables changes, which are more "user-facing"